flink-tsfile-connector/src/test/java/org/apache/iotdb/flink/util/TsFileWriteUtil.java

```diff
@@ -149,16 +148,15 @@ public static void create2(String tsfilePath) throws Exception {
 		TSRecord tsRecord = new TSRecord(9, "device_1");
 		DataPoint dPoint1 = new FloatDataPoint("sensor_1", 1.2f);
 		DataPoint dPoint2 = new IntDataPoint("sensor_2", 20);
-		DataPoint dPoint3;
-		tsRecord.addTuple(dPoint1);
+        tsRecord.addTuple(dPoint1);
```

Comment:
this format may have some error

Reply:
Indeed, the entire TsFileWriteUtil is indented with tabs. I have changed the tab indentation to space indentation.